### PR TITLE
Initial

### PR DESCRIPTION
**Pull request by Aiksterzh**
- initial file setup
- fix endpoints
- set up postgres schema

### CODE REVIEW

File: database/postgress.sql [hunk 1]
```diff
@@ -0,0 +1,65 @@
+-- DROP DATABASE IF EXISTS booking;
+
+-- CREATE DATABASE booking;
+
+-- USE booking;
+
+DROP TABLE IF EXISTS users;
+
+CREATE TABLE users (
+  id BIGINT NOT NULL,
```
Comment (vlad-pisotskyi) [inline database/postgress.sql]: I think it's BIGSERIAL

File: database/postgress.sql [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  userName VARCHAR(100) NOT NULL,
+  creditCardNumber BIGINT,
+  billingAddress VARCHAR(600),
+  PRIMARY KEY (id)
+);
+
+DROP TABLE IF EXISTS owners;
+
+CREATE TABLE owners (
+  id BIGINT NOT NULL,
```
Comment (vlad-pisotskyi) [inline database/postgress.sql]: BIGSERIAL

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+# Coverage directory used by tools like istanbul
+coverage
+
+# Grunt intermediate storage (http://gruntjs.com/creating-plugins#storing-task-files)
+.grunt
+
+# Compiled Dirs (http://nodejs.org/api/addons.html)
+build/
+dist/
```
Comment (vlad-pisotskyi) [inline .gitignore]: Newline required at end of file

File: client/src/components/Booking.jsx [hunk 4]
```diff
@@ -145,7 +145,8 @@ class Booking extends Component {
   }
 
   checkoutWithTripDetails(tripDetails) {
-    axios.post('/booking', tripDetails)
+    let id = window.location.pathname;
```
Comment (vlad-pisotskyi) [inline client/src/components/Booking.jsx]: you can put this in props, not to repeat yourself.
(i know its STC not FEC) XD

**Review by Pisotski — COMMENTED** [review]
I am not sure about BIGSERIAL (I didn't check myself).
Looks good

**Review by Pisotski — APPROVED** [review]
looks good

**Pull request merged**